nucliadb/src/nucliadb/common/maindb/pg.py

```diff
@@ -44,13 +44,32 @@
 );
 """
 
+logger = logging.getLogger(__name__)
 
+# Request Metrics
 pg_observer = metrics.Observer(
     "pg_client",
     labels={"type": ""},
 )
 
-logger = logging.getLogger(__name__)
+# Pool metrics
+POOL_METRICS_COUNTERS = {
+    # Requests for a connection to the pool
+    "requests_num": metrics.Counter("pg_client_pool_requests_total"),
+    "requests_queued": metrics.Counter("pg_client_pool_requests_queued_total"),
+    "requests_errors": metrics.Counter("pg_client_pool_requests_errors_total"),
+    "requests_wait_ms": metrics.Counter("pg_client_pool_requests_queued_seconds_total"),
+    "usage_ms": metrics.Counter("pg_client_pool_requests_usage_seconds_total"),
+    # Pool opening a connection to PG
```

Comment:
maybe you want to rename some of the metrics.Counter, as internally adds the trailing `_count` already

Reply:
Yeah, I know it adds `_total` if it's not there. However I find that having the same name here in the code as the metric in prometheus makes it easier to search for things. It would not be the first time I put the prometheus name into the code search and found nothing because something was being added to the name automatically.